src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -86,8 +86,7 @@ Class {
 		'extensionToolsNotebook',
 		'toolbarCommandGroup',
 		'debuggerActionModel',
-		'unsavedCodeChanges',
-		'hasProceeded'
+		'unsavedCodeChanges'
 	],
```

Comment:
This variable is removed because it is not used?
I remember Pablo introduced it to fix a problem with spec when closing the window, but I don't remember if we redesigned it after and if that variable is just a remain.

Reply:
It got removed by accident.
Iceberg showed a ton of changes I didn't make.
I probably failed to uncheck this one when doing the commit.
I pushed another commit to re-add this variable.

Reply:
ok, then for me we can merge it now. thanks!

Reply:
(just waiting to see if the tests pass, but I think they never do on this repo I do not know why)